tutorials/forward/25_automated_coreg.py

```diff
@@ -26,8 +28,8 @@
 from mne.coreg import Coregistration
 from mne.io import read_info
 
-
 data_path = mne.datasets.sample.data_path()
+# data_path and all paths built from it are `pathlib.Path`_ objects
```

Comment:
I don't think this will render because it's in a code comment :(

Plus I think this isn't specific/relevant enough to the automated coreg stuff to include here, maybe somewhere in `datasets` instead if you want?
```suggestion
```